### PR TITLE
Pushes input inside the bottom group - login.

### PR DESCRIPTION
**Pull request by raghunayyar**
Current the input on the login page is dangling outside the bottom group. I possible fix for https://github.com/owncloud/core/issues/18629

@owncloud/designers @jancborchardt @MorrisJobke @davitol

### CODE REVIEW

**Comment by MorrisJobke** [comment]
Tested and works :+1:

**Comment by davitol** [comment]
:-1: It stills not working for me, but as I said in the issue it only happens in my cursed laptop

**Comment by MorrisJobke** [comment]
This is because the form element on @davitol 's PC. I guess this is because the form is defined as `em` and not in px.

```
#body-login form {
	position: relative;
	width: 22em;
	margin: 2em auto 2em;
	padding: 0;
}
```
https://github.com/owncloud/core/blob/master/core/css/styles.css#L481-L486

@jancborchardt Can we move this to px?

**Comment by raghunayyar** [comment]
This anyway should go in, considering the semantics. cc @jancborchardt

**Comment by jancborchardt** [comment]
@raghunayyar can you maybe just try to convert the em to px to fix that additional issue?

**Comment by MorrisJobke** [comment]
> @raghunayyar can you maybe just try to convert the em to px to fix that additional issue?

... and rebase please

**Comment by raghunayyar** [comment]
rebased. cc @MorrisJobke and made the em to px conversion as well. Please check @davitol

File: core/css/styles.css [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/* Copyright (c) 2011, Jan-Christoph Borchardt, http://jancborchardt.net
+f/* Copyright (c) 2011, Jan-Christoph Borchardt, http://jancborchardt.net
```
Comment (MorrisJobke) [inline core/css/styles.css]: :boom:

Comment (jancborchardt) [inline core/css/styles.css]: @raghunayyar can you fix this and then squash the commits into one? Otherwise it looks fine.

**Comment by raghunayyar** [comment]
Done, this should be merge ready now. @MorrisJobke

**Comment by jancborchardt** [comment]
@davitol please review.

@raghunayyar please don’t call the one commit »squashing all changes into one« ;) – and rebase

**Comment by davitol** [comment]
:+1: The PR fixes #18629

**Comment by DeepDiver1975** [comment]
:+1:

**Pull request merged**